bot/cogs/error_handler.py

```diff
@@ -170,7 +170,7 @@ async def handle_user_input_error(self, ctx: Context, e: errors.UserInputError)
             await prepared_help_command
             self.bot.stats.incr("errors.too_many_arguments")
         elif isinstance(e, errors.BadArgument):
-            await ctx.send(f"Bad argument: {e}\n")
+            await ctx.send("Bad argument: Please double check your input arguments and try again.\n")
```

Comment:
```suggestion
            await ctx.send("Bad argument: Please double-check your input arguments and try again.\n")
```
It seems the more correct form is to hyphenate here.